src/style.css

```diff
@@ -1,3 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  max-width: 1200px;
+  min-width: 1024px;
+  padding: 0 50px;
+  /* text-align: right; */
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
```

Comment:
center your block horizontally. There is a lot of space on the right
<img width="1725" alt="Screenshot 2023-06-29 at 08 53 30" src="https://github.com/mate-academy/layout_moyo-header/assets/135232001/d0730a35-6b6c-41e1-869a-aa4f12b35c7e">